projects/productconfig/src/variant-configurator/overview.module.ts

```diff
@@ -0,0 +1,120 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { RouterModule } from '@angular/router';
+import { NgSelectModule } from '@ng-select/ng-select';
+import {
+  CmsConfig,
+  ConfigModule,
+  I18nModule,
+  UrlModule,
+  UserService,
+} from '@spartacus/core';
+import {
+  CmsPageGuard,
+  ConfigAddToCartButtonComponent,
+  ConfigOverviewFormComponent,
+  ConfigPriceSummaryComponent,
+  ConfigTabBarComponent,
+  GenericConfiguratorModule,
+  IconModule,
+  PageLayoutComponent,
+} from '@spartacus/storefront';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    GenericConfiguratorModule,
+
+    RouterModule.forChild([
+      {
+        path:
+          'configureOverviewCPQCONFIGURATOR/:ownerType/entityKey/:entityKey/displayOnly/:displayOnly',
+        component: PageLayoutComponent,
+        data: { pageLabel: '/configureOverviewCPQCONFIGURATOR' },
+        canActivate: [CmsPageGuard],
+      },
+      {
+        path:
+          'configureOverviewCPQCONFIGURATOR/:ownerType/entityKey/:entityKey',
+        component: PageLayoutComponent,
+        data: { pageLabel: '/configureOverviewCPQCONFIGURATOR' },
+        canActivate: [CmsPageGuard],
+      },
+    ]),
+    ConfigModule.withConfig(<CmsConfig>{
+      cmsComponents: {
+        VariantConfigurationTabBar: {
+          component: ConfigTabBarComponent,
+          guards: [],
+        },
+        VariantConfigurationOverview: {
+          component: ConfigOverviewFormComponent,
+          guards: [],
+        },
+        VariantConfigurationPriceSummary: {
+          component: ConfigPriceSummaryComponent,
+          guards: [],
+        },
+        VariantConfigurationAddToCartButton: {
+          component: ConfigAddToCartButtonComponent,
+          guards: [],
+        },
+      },
+      layoutSlots: {
```

Comment:
I see some duplication in this and in `projects/productconfig/src/variant-configurator/interactive-configuration.module.ts`. 
Maybe we can create one entry module `variant-configurator.module.ts` and specify the common stuff in it? This entry module _can_ have `forRoot` method (similar to https://github.com/SAP/spartacus/blob/27dfc18222d4a924993e0569e78cfa07ce8927a7/projects/cds/src/cds.module.ts#L18). This module will be imported by clients when they want to enable the _variant-configurator_ feature.
What do you think?

Reply:
Yes, sounds like a good idea